test/unit/builtins/string.spec.ts

```diff
@@ -56,11 +56,11 @@ test.each([
     { inp: "hello test", searchValue: "test", replaceValue: "" },
     { inp: "hello test", searchValue: "test", replaceValue: "world" },
     { inp: "hello test", searchValue: "test", replaceValue: "%world" },
+    { inp: "hello test", searchValue: " test", replaceValue: "." },
     { inp: "hello %test", searchValue: "test", replaceValue: "world" },
     { inp: "hello %test", searchValue: "%test", replaceValue: "world" },
-    { inp: "hello test", searchValue: "test", replaceValue: (): string => "a" },
-    { inp: "hello test", searchValue: "test", replaceValue: (): string => "%a" },
-    { inp: "aaa", searchValue: "a", replaceValue: "b" },
```

Comment:
Why did you remove this? I don't think there is another test that replaces multiple references in the same string.